Source/Blazorise/Components/Table/Table.razor.cs

```diff
@@ -139,6 +139,21 @@ private async ValueTask RecalculateResize()
             }
         }
 
+        /// <summary>
+        /// If table has FixedHeader. 
+        /// Invokes javascript function moving the table scroll position to the provided pixels.
+        /// </summary>
+        /// <param name="pixels"></param>
+        /// <returns></returns>
+        public ValueTask FixedHeaderScrollTableTo(int pixels )
```

Comment:
This migth be usefull for regular table. Not just for fixed header. What do you think we enable it for all and rename it into ScrollTo

Reply:
I'm not on desktop now... But Didn't I implement it on table?

Reply:
Oh regular table... But does regular table have scrollbar? That should only be avaibake on fixed header?

Reply:
You're right. It's not working for the regular table. I have just tested it. 

We can try adding this feature https://stackoverflow.com/a/7853216/833106. I think it might be good for longer tables.

Reply:
Oh. I didn't know about that feature. That might be a good one yes. Let's try.
Might have both a ScrollToPx and Scroll To Element? 
I'll let you know how it goes once I try it.

Reply:
Yep, I have created a new ticket. Once i merge this you can try making it. Shouldn't be a problem I think.